ajax/libs/proteic/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "proteic",
+  "filename": "proteic.js",
+  "version": "0.0.27",
+  "description": "Streaming and static data visualization for the modern web",
+  "homepage": "https://proteic.js.org/",
+  "keywords": [
+    "d3",
+    "chart",
+    "data visualization"
+  ],
+  "author": {
+    "name": "Treelogic S.L.",
+    "url": "http://www.treelogic.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/proteus-h2020/proteic.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/proteus-h2020/proteic.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(lighter*)"
```

Comment:
Let's modify to `"**/!(lighter*|*.node.*)"` for excluding `proteic.node.js` in v0.0.4~v0.0.6.
Thanks.